app/controllers/peer_reviews_controller.rb

```diff
@@ -63,13 +63,13 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_peer_review
-      @peer_review = PeerReview.find(params[:id])
-    end
```

Comment:
Do you want this undone? Looks like rubymine auto-un-indented when converting back to the old version.